sqlmodel/main.py

```diff
@@ -506,6 +528,9 @@ def __init__(
         base_is_table = any(is_table_model_class(base) for base in bases)
         if is_table_model_class(cls) and not base_is_table:
             for rel_name, rel_info in cls.__sqlmodel_relationships__.items():
+                if rel_name in cls.__sqlalchemy_constructs__:
+                    # Skip hybrid properties
+                    continue
```

Comment:
Seems to me like members of `__sqlalchemy_constructs__` are not added to  `__sqlmodel_relationships__`, so this check is redundant ? (otherwise the comment is missing column properties and declared attrs)

Reply:
A safety net against unforeseen attributes. You could try deleting